### ISSUE TITLE
MD051 doesn't recognize capital accent characters

### ISSUE THREAD

**Issue by nschonni** [DavidAnson/markdownlint#536]
I had the following, and originally thought it was the accented characters, but it looks like it's actually the fact that Markdownlint keeps the `'`, but GitHub strips it in the auto-anchor

```md
([Français](#données-pour-lÉchange-de-ressources-ouvert))

Text

# Données pour l'Échange de ressources ouvert
```
Live version https://github.com/canada-ca/ore-ero/blob/master/_data/README.md

**Comment by DavidAnson** [comment]
Remember that GitHub lower-cases every character when making the link for a heading. Your sample still has a capital letter (`É`) - lower-casing it passes the rule as currently implemented: https://dlaa.me/markdownlint/#%25m%23%20Issue%20536%0A%0A%23%23%20Donn%C3%A9es%20pour%20l'%C3%89change%20de%20ressources%20ouvert%0A%0A%5BFran%C3%A7ais%5D(%23donn%C3%A9es-pour-l%C3%A9change-de-ressources-ouvert)%0A

There are probably a few different ways this could work, but the all-lower-case the rule enforces now reflects what shows up in the browser URL (in Chrome) and seems like a reasonable canonical implementation.

**Comment by nschonni** [comment]
Yeah, looks like I was missreading the double encoding characters as the apostrophe then the accent character, but it was just the accented character. I know this rule gets a little fuzzy since it isn't a CommonMark spec'd thing.
PS: also ran into an issue with the MDN content repo because their anchor generator behaves differently, but I haven't come up with an option proposal yet
